src/tools/interop/idt/discovery/__init__.py

```diff
@@ -0,0 +1,23 @@
+#
+#    Copyright (c) 2023 Project CHIP Authors
+#    All rights reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+from .matter_ble import MatterBleScanner
+from .matter_dnssd import MatterDnssdListener
+
+__all__ = [
```

Comment:
Can you make the Discovery modules adhere to a common interface?

Reply:
I opened an issue to track the remaining issues and added your comment above and this one into the issue. I'm going to close this off for now so we can get this PR merged, but the issue tracker is here: https://github.com/project-chip/connectedhomeip/issues/29477